tests/unit/email/test_init.py

```diff
@@ -2057,6 +2066,7 @@ def test_send_removed_project_release_file_email_to_maintainer(
             version="0.0.0",
             project=pretend.stub(name="test_project"),
             created=datetime.datetime(2017, 2, 5, 0, 0, 0, 0),
+            yanked_reason="",
```

Comment:
I had to add `yanked_reason` to all the stubs in this file, with the alternative being to use `getattr(release, "yanked_reason")` in `warehouse/email/__init__.py`, which felt less clean 🤔